File: C++/Easy/WrongAnswer/circular-sentence_5.cpp
class Solution {
public:
    bool isCircularSentence(string sentence) {
        
        int i = 0;
        int j = 0;
        int n = sentence.size();
        while (i < n) {
            while (i < n && sentence[i] != ' ') {
                ++i;
            }
            if (sentence[i - 1] != sentence[j]) {
                return false;
            }
            ++i;
            j = i;
        }
        return true;
    }
};

File: Python3/Easy/WrongAnswer/circular-sentence_5.py
class Solution:
    def isCircularSentence(self, sentence: str) -> bool:
        
        sentence = sentence.split(' ')
        return all([sentence[i][-1].lower() == sentence[i+1][0].lower() for i in range(len(sentence)-1)]) and sentence[-1][-1].lower() == sentence[0][0].lower()